src/main/java/com/birbit/jsonapi/JsonApiDeserializer.java

```diff
@@ -58,17 +69,19 @@ public JsonApiResponse deserialize(JsonElement json, Type typeOfT, JsonDeseriali
 
         JsonApiLinks links = parseLinks(context, jsonObject);
 
+        JsonApiMeta meta = parseMeta(context, jsonObject);
```

Comment:
Meta is part of the JSON API spec and we started to use it for delivery times response.